code/modules/economy/requisition/rc_scientific.dm

```diff
@@ -545,7 +545,7 @@ ABSTRACT_TYPE(/datum/rc_entry/item/organ)
 			if(PROTOTYPIST_ENGINEER)
 				prototypist_desc = "Engineering firm"
 				if(prob(60))
-					src.rc_entries += rc_buildentry(/datum/rc_entry/item/tscan,rand(2,4))
+					src.rc_entries += rc_buildentry(/datum/rc_entry/item/tscan,rand(1,2))
```

Comment:
these can always be filled by 1 electrical maintenance crate

Reply:
They can, yes. Just less tedium to not randomly need *four* of them.